.changeset/violet-beans-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+"@commercetools-uikit/spacings-inline": patch
+---
+
+docs(spacings-inline): flex-wrap alternative
```

Comment:
```suggestion
Document recommendation for [`<Grid />`](https://uikit.commercetools.com/?path=/story/components-grid--grid) instead of using `flex-wrap`.
```

Reply:
My bad 😅 .